docs/advanced/general-privacy.md

```diff
@@ -0,0 +1,63 @@
+
+Many times this is something physical, for example leaving behind a printed QR code somewhere, writing our nickname or real name on a piece of paper. Your name can be inprinted on the next page of the sticky notes stack, for example. Or that could also happen, that you leave behind your phone or computer unlocked for some minutes, with important information displayed on screen.
+
+## Centralized mixers
+
+The aim of a cryptocurrency mixer is to break the transaction graph. An old version of cryptocurrency mixers is the centralized mixers (also called scrambler, thumbler, shuffler). A user would send in Bitcoin or other cryptocurrency to a mixing service (that can be a website), and he or she will receive back coins that were mixed with the coins of the other users. They will charge a fee. The downside of this, is that the mixer could keep logs of the mixed coins. Other possibility is that the mixer will steal the coins. Using an online casino, poker site or exchange, that does not require identification, creates more or less the same result as using a mixer.
+Either we use a centralized mixer or the later mentioned solutions, [amount correlation](https://en.bitcoin.it/wiki/Privacy#Amount) is a problem. If we send in 1.435 BTC to a mixer, and someone receives 1.434 BTC that's a quite strong indication that the transaction belongs to us.
+Amount correlation will be a problem even if we use a so called _black box privacy technique_, like sending in our money to an anonymous exchange, exchanging it to a privacy coin which uses [zero-knowledge proof](https://en.wikipedia.org/wiki/Zero-knowledge_proof), and then switching back the money to our original currency. Amount correlation can still lead to loss of privacy. One can address this issue by breaking up the total balance to random chunks and distributing it to newly generated addresses, over a longer period of time.
+
+## Non-custodial coin mixing
+
+It is possible to mix coins without a centralized exchange. One method is [CoinJoin](https://en.bitcoin.it/wiki/CoinJoin), which will combine multiple payments from multiple spenders into a single transaction.
+Other method is [CoinSwap](https://en.bitcoin.it/wiki/CoinSwap), which is using atomic swaps. A third method is [TumbleBit](https://en.bitcoin.it/wiki/TumbleBit), which allows parties to make payments through an untrusted Tumbler.
+[Decred's build in coin mixing](https://blog.decred.org/2019/08/28/Iterating-Privacy/) feature is based on the CoinShuffle++ protocol.
+
+## Using the Lightning Network for privacy
+
+Privacy attacks like [common-input-ownership heuristic](https://en.bitcoin.it/wiki/Privacy#Common-input-ownership_heuristic), [address reuse](https://en.bitcoin.it/wiki/Privacy#Address_reuse) and change address detection fundamentally don’t work on any off-chain transactions technology. The Lightning Network is an off-chain transaction technology. One possible downside is that the operator of the lightning node could keep logs.
```

Comment:
As a reader, I am wondering what information can a lightning node log, and why is that a privacy risk?